src/System/useFeatureFlag.tsx

```diff
@@ -47,6 +48,22 @@ export function useFeatureVariant(featureName: string): Variant | null {
   return variant
 }
 
+export const getFeatureVariant = (featureName: string): Variant | null => {
+  const featureFlags = getENV("FEATURE_FLAGS")
+  const variant = featureFlags?.[featureName]?.variant
+
+  if (!variant) {
+    console.error(
+      "[Force] Error: cannot find variant on featureFlags: ",
+      featureFlags
+    )
+
+    return null
+  }
+
+  return variant
```

Comment:
Question - should you be checking whether the feature flag is enabled before returning a variant? This might mean that a created feature flag (but not enabled), would still return a variant here, thus forcing behavior away from the 'control' or default even when the flag is not enabled (but does exist).

Reply:
I see what you're saying though in practice when I've disabled it it doesn't return at all.

Reply:
Just to be clear, if you have a user or group of users that are not being served the feature (i.e., they don't fall into the group in the activation strategy), then the feature flag would have the below shape, so there _would_ be a variant here. If the toggle itself is disabled then the feature isn't set on the `FEATURE_FLAG` object. 

```js
{ 
   exampleFlag: {
      flagEnabled: false,
      variant: {
         enabled: false,
         name: "disabled"
      }
   }
}
```

Reply:
See https://github.com/artsy/force/pull/11550